test/api_integration/apis/telemetry/telemetry_last_reported.ts

```diff
@@ -25,10 +25,7 @@ export default function optInTest({ getService }: FtrProviderContext) {
     });
 
     it('GET should return undefined when there is no stored telemetry.lastReported value', async () => {
-      await supertest
-        .get('/api/telemetry/v2/last_reported')
-        .set('kbn-xsrf', 'xxx')
-        .expect(200, { lastReported: undefined });
```

Comment:
For some reason, now, when reporting `lastReported: undefined` the serializer removed the field entirely... Functionally, it's the same for JS, so no biggie.